src/Simplex/Chat.hs

```diff
@@ -1748,6 +1742,12 @@ cleanupManager user = do
       timedItems <- withStore' $ \db -> getTimedItems db user startTimedThreadCutoff
       forM_ timedItems $ uncurry (startTimedItemThread user)
 
+startProximateTimedItemThread :: ChatMonad m => User -> (ChatRef, ChatItemId) -> UTCTime -> m ()
+startProximateTimedItemThread user itemRef deleteAt = do
+  ts <- liftIO getCurrentTime
+  when (diffInMicros deleteAt ts <= cleanupManagerInterval * 1000000) $
```

Comment:
add diffInSeconds to Util, the second argument won't have to be converted

``` haskell
diffInSeconds :: UTCTime -> UTCTime -> Int
diffInSeconds a b = (`div` 1000000000000) $ diffInPicos a b
```

Reply:
Why? Even diffInMicros is unnecessary, you could just use the fact that it can be converted to seconds